egs/timit_ssc/ssr1/local/featprepare.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+if __name__ == "__main__":
+    for dataset in ["data/train/", "data/test/"]:
```

Comment:
This is a little bit hard-corded with a strong assumption for these directory structures, but it's OK. This is just a note.

Reply:
Many thanks for your help!
I'd like to fix it anytime if  necessary!